ROS/osr/src/roboclaw_wrapper.py

```diff
@@ -39,10 +39,10 @@ def __init__(self):
         self.corner_max_vel = 1000
         self.corner_accel = 2000
         self.roboclaw_overflow = 2**15-1
+        # drive motor acceleration
         accel_max = 655359
```

Comment:
Should this be 2^16 = 65536?

Reply:
I see where this came from: https://resources.basicmicro.com/understanding-acceleration-and-deceleration-values/